### PR TITLE
InstructorHomePageUiTest and InstructorFeedbackEditCopyUiTest failing #5032

### LINKED ISSUE #5032: InstructorHomePageUiTest and InstructorFeedbackEditCopyUiTest failing
Fails on wait for id:copiedfsname

### PR DESCRIPTION
**Pull request by LowWeiLin**
Fixes #5032

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/AppPage.java [hunk 1]
```diff
@@ -199,6 +199,15 @@ public void waitForPageToLoad() {
         browser.selenium.waitForPageToLoad(TestProperties.inst().TEST_TIMEOUT_PAGELOAD);
     }
     
+    public void waitForElementNotCovered(final WebElement element){
```
Comment (kanghj) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: Header comments

File: src/test/java/teammates/test/pageobjects/AppPage.java [hunk 2]
```diff
@@ -718,6 +727,19 @@ public boolean isElementSelected(String elementId) {
...
+    /**
+     * This function checks if the midpoint of an element is covered by any other element.
+     * @param element
+     * @return true if element is covered, false otherwise.
+     */
+    public boolean isElementCovered(WebElement element) {
+        int x = element.getLocation().x + element.getSize().width / 2;
+        int y = element.getLocation().y + element.getSize().height / 2;
+        JavascriptExecutor js = (JavascriptExecutor) browser.driver;
+        WebElement topElem = (WebElement) js.executeScript("return document.elementFromPoint("+x+","+y+");");
```
Comment (kanghj) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: spaces before and after the <code>+</code>

File: src/test/java/teammates/test/pageobjects/AppPage.java [hunk 3]
```diff
@@ -718,6 +727,19 @@ public boolean isElementSelected(String elementId) {
             return false;
         }
     }
+    
+    /**
+     * This function checks if the midpoint of an element is covered by any other element.
+     * @param element
+     * @return true if element is covered, false otherwise.
+     */
+    public boolean isElementCovered(WebElement element) {
```
Comment (kanghj) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: should <code>element</code> be final?

File: src/test/java/teammates/test/pageobjects/AppPage.java [hunk 4]
```diff
@@ -718,6 +727,19 @@ public boolean isElementSelected(String elementId) {
             return false;
         }
     }
+    
+    /**
+     * This function checks if the midpoint of an element is covered by any other element.
```
Comment (kanghj) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: Can remove <code>This function</code>

File: src/test/java/teammates/test/pageobjects/AppPage.java (hunk 3 above)
Comment (LowWeiLin) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: No

**Comment by thyageshm** [comment]
@LowWeiLin here too

**Pull request merged**